node/cn/backend.go

```diff
@@ -228,6 +228,7 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 		MaxBlockGasUsedForBaseFee: governance.MaxBlockGasUsedForBaseFee(),
 		BaseFeeDenominator:        governance.BaseFeeDenominator(),
 	}
+	chainConfig.Governance.Reward.Kip82Ratio = governance.Kip82Ratio()
```

Comment:
I think It's better to comment something like "we can assure Governance and Reward is not nil", also at L224

Reply:
@kjeom FYI, these lines will be removed by https://github.com/klaytn/klaytn/pull/1623/files#diff-de2bd0674bf71cd6e3d92cb5621577831308e4cc990a15933e5e853f3358e4e0L224.
How about we make a comment on the above at https://github.com/klaytn/klaytn/pull/1623/files#diff-de2bd0674bf71cd6e3d92cb5621577831308e4cc990a15933e5e853f3358e4e0R218?

Reply:
Ohh, then we don't need to comment it because we can assure these are not nil by [default chainConfig](https://github.com/ian0371/klaytn/blob/dee2c1b22793a97ac31110d90f86970618cb7e82/params/config.go#L411)